package.json

```diff
@@ -22,10 +22,12 @@
   },
   "homepage": "https://github.com/daoneandonly/backend-a2#readme",
   "dependencies": {
+    "body-parser": "^1.19.0",
     "concurrently": "^6.0.0",
     "dotenv": "^8.2.0",
     "ejs": "^3.1.6",
     "express": "^4.17.1",
+    "express-rate-limit": "^5.2.6",
```

Comment:
Using `express-rate-limit` is an excellent idea. However the code you're adding below is not really a ratelimiter. We want to add this to the _server_ instead of at the _client's webbrowser_. You can check the little documentation there is on [express-rate-limit](https://www.npmjs.com/package/express-rate-limit). 

Note that we need to define our values, and then apply it to our app using `app.use(...)`. Now a rate limiter can also hinder us if we set it on `GET` requests, meaning only a `X` times a second a `GET` request can be preformed on our webserver.

Let's apply this `express-rate-limit`  in our `app.js`, on at least on of our `POST` requests.

Reply:
In the first paragraph of the documentation of `express-rate-limit` they say it is rate limiting middleware for express. So this means it works on the server. The countdown works only on the browser to make it easier to understand for the user.

Reply:
Completely missed your changes in `app(..)`, my bad! Looks pretty cool. Be sure to merge the `develop` branch into this.

Reply:
I actually made a new pull request so i'll close this one!